indico/modules/events/registration/templates/display/participant_list.html

```diff
@@ -11,6 +11,14 @@
     {% endif %}
 {% endblock %}
 
+{% block description %}
+    {%- if tables|length > 1 -%}
+        {% trans -%}
+            The lists of participants grouped by the registration form they used to registered to the event.
```

Comment:
That's basically what I mean - for a *user* who cares about the list of participants it does not really matter that it's based on data coming from registration forms. It's a list of people participating in the event, possibly grouped by let's say "participants" and "exhibitors"

Anyway, let's get a third opinion 
ping @pferreir